### PR TITLE
Teaser tweaks - list and slider teasers.

### PR DESCRIPTION
**Pull request by rasben**
**Introduce list-teaser viewmode for media, without caption. DDFFORM-465**

This is to avoid that the caption shows up, which looks weird in teaser lists - and also breaks the design.

------

**Dont display created date on page teaser. DDFFORM-524**

After preference from DDF.

https://reload.atlassian.net/browse/DDFFORM-524

https://reload.atlassian.net/browse/DDFFORM-465

### CODE REVIEW

File: config/sync/core.entity_view_display.eventinstance.default.list_teaser.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ content:
     weight: 11
     region: content
   event_description:
-    type: text_default
+    type: basic_string
```
Comment (Dresse) [inline config/sync/core.entity_view_display.eventinstance.default.list_teaser.yml:80]: Is this change intentional? Didn't see it mentioned in the PR description.

**Review by Dresse — APPROVED** [review]
LGTM, but I have a small comment/question.

File: config/sync/core.entity_view_display.eventinstance.default.list_teaser.yml (hunk 1 above)
Comment (rasben) [reply config/sync/core.entity_view_display.eventinstance.default.list_teaser.yml:80]: Yes and no - It's not something i did actively, but, it has come from saving the view display, and apparantly this config was missing :) I dont see it as a problem, but good that you're awake!
